src/contracts/interfaces/IEigenPod.sol

```diff
@@ -50,6 +50,17 @@ interface IEigenPod {
     }
 
 
+    struct ProofService {
+        address caller;
+        // whether or not the proof fulfiller is allowed to fulfill proofs
+        bool permission;
+        // whether or not the proof fulfiller has been added
+        uint256 maxFee;
```

Comment:
i feel like this should be in the user's signature?

Reply:
it is?